lib/vehicle.rb

```diff
@@ -6,13 +6,15 @@ class Vehicle
               :make,
               :model,
               :engine
+  attr_accessor :registration_date
 
   def initialize(vehicle_details)
     @vin = vehicle_details[:vin]
     @year = vehicle_details[:year]
     @make = vehicle_details[:make]
     @model = vehicle_details[:model]
     @engine = vehicle_details[:engine]
+    @registration_date = nil
```

Comment:
You should update your vehicle_spec to test that this attribute is present as well as testing the `plate_type` method below.